rest/test_results_routes.go

```diff
@@ -15,7 +15,7 @@ import (
 
 ///////////////////////////////////////////////////////////////////////////////
 //
-// GET /testresults/task_id/{task_id}
+// GET /test_results/task_id/{task_id}
```

Comment:
Doesn't the REST service route also have to be changed?

Reply:
I am going to change it in another ticket, I don't want to break too many things right now.

Reply:
I thought nothing was using the test results routes right now. What's wrong with changing it?

Reply:
Staging, but I will change it before I merge staging. It just requires me to also change timber and revendor and I am trying to debug something else before that.